pylink_utils/get_secret.py

```diff
@@ -0,0 +1,67 @@
+import base64
+import json
+import os
+from typing import Optional
+
+import boto3
+from botocore.exceptions import ClientError
+
+
+def get_secret(secret_name: str, region_name: str, profile: Optional[str] = None) -> [dict, bytes]:
```

Comment:
Return type should be `Union[dict, bytes]`

Reply:
Currently it means a list with two items, the 0 index is a dict and the 1 is bytes